From b62b3dc970db05abaad25ec3249eaa1c203e42bf Mon Sep 17 00:00:00 2001
From: Tsukasa OI <research_trasio@irq.a4lg.com>
Date: Wed, 6 Sep 2023 02:35:52 +0000
Subject: [PATCH 17/30] RISC-V: typo: add closing paren to a comment

gcc/ChangeLog:

	* config/riscv/zicond.md: Add closing parent to a comment.
---
 gcc/config/riscv/zicond.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gcc/config/riscv/zicond.md b/gcc/config/riscv/zicond.md
index 1721e1011ea..c28bee5d570 100644
--- a/gcc/config/riscv/zicond.md
+++ b/gcc/config/riscv/zicond.md
@@ -64,7 +64,7 @@
 )
 
 ;; Combine creates this form in some cases (particularly the coremark
-;; CRC loop.
+;; CRC loop).
 (define_split
   [(set (match_operand:X 0 "register_operand")
 	(and:X (sign_extract:X (match_operand:X 1 "register_operand")
-- 
2.25.1

